qiskit/providers/ibmq/ibmqjob.py

```diff
@@ -150,11 +144,11 @@ def __init__(self, backend, job_id, api, qobj=None,
             # enough information to recreate the `Job`. If that is the case, try
             # to make use of that information during instantiation, as
             # `self.status()` involves an extra call to the API.
-            if api_status == 'VALIDATING':
+            if api_status == ApiJobStatus.VALIDATING.value:
```

Comment:
For `ApiJobStatus` I'm not too concerned as it is only used by the internals of the `IBMQProvider`, and the consumers are "under control" - perhaps your comment was geared towards `JobStatus` (or perhaps I'm reading it wrong altogether!)? It would indeed be more related to terra than to the provider itself, as that's where `JobStatus` lives at the moment.

Reply:
Right. I did not realize we were the only consumers. Not blocking on this.